server/http_handler_test.go

```diff
@@ -690,7 +690,7 @@ func (ts *HTTPHandlerTestSuite) TestPprof(c *C) {
 		}
 		time.Sleep(time.Millisecond * 10)
 	}
-	log.Fatalf("Failed to get profile for %d retries in every 10 ms", retryTime)
+	zaplog.Fatal("Failed to get profile for %d retries in every 10 ms", zap.Int("retryTime", retryTime))
```

Comment:
Why use zaplog instead of replacing logrus here?

Reply:
Now we use `zaplog` instead of `logrus`. But `slow log` has not updated this log yet, so there are two logs in this package.

Reply:
@jackysp